assets/scss/_variables_project.scss

```diff
@@ -8,20 +8,16 @@ $secondary: #326ce5;
 }
 
 .td-navbar .navbar-brand svg {
-	height: 70px !important;
-	width: 70px;
+	height: 60px !important;
+	width: 60px;
 }
 
-.td-toc {
+.td-sidebar-toc {
 	padding-top: 1.75rem !important;
 }
 
 @media (min-width: 768px) {
-	.td-sidebar {
-		padding-top: 4.5rem !important;
-	}
-
 	.td-main main {
-		padding-top: 7.5rem !important;
+		padding-top: 6rem !important;
```

Comment:
Please explain these CSS changes.

Reply:
The td-sidebar-toc change is needed because, the html was also changed, because the upstream html was changed (see layouts/docs/baseof.html).

The logo size I changed because it looks better on my machine:
![image](https://user-images.githubusercontent.com/42113979/123265880-c9a75a80-d4fb-11eb-9d5d-24a2e9e0c6e8.png)
old (left) vs new (right)

Please let me know if I should revert the logo change.

Reply:
I like it.